src/main/java/com/group12/LaunchInterceptorConditionCollection.java

```diff
@@ -594,4 +644,23 @@ private double angleFromThreePoints(Point vertex, Point point1, Point point2) {
         double magb = Math.sqrt(Math.pow(bx, 2) + Math.pow(by, 2));
         return Math.acos((ax * bx + ay * by) / (maga * magb));
     }
+
+    private boolean threePointsFulfillLIC9(Point v, Point p1, Point p2, double epsilon){
```

Comment:
It is a bit strange to have a special function for LIC9 only. You can generalize the logic and modify LIC2 to use this function as well. LIC2 is merged in #38

Reply:
yeah its from trying to reduce code complexity